src/Kiota.Builder/Writers/Go/GoConventionService.cs

```diff
@@ -11,44 +12,39 @@ public class GoConventionService : CommonLanguageConventionService
 
         public override string DocCommentPrefix => string.Empty;
 
-        public override string PathSegmentPropertyName => "pathSegment";
-
-        public override string CurrentPathPropertyName => "currentPath";
-
-        public override string HttpCorePropertyName => "httpCore";
-
+        private const string PathSegmentPropertyName = "pathSegment";
+        private const string CurrentPathPropertyName = "currentPath";
+        private const string HttpCorePropertyName = "httpCore";
         public override string ParseNodeInterfaceName => "ParseNode";
-        public override string RawUrlPropertyName => "isRawUrl";
-
+        #pragma warning disable CA1822 // Method should be static
         public object AbstractionsHash => "ida96af0f171bb75f894a4013a6b3146a4397c58f11adb81a2b7cbea9314783a9";
```

Comment:
Any reason this can't be a `const string`?

Reply:
```suggestion
        public string AbstractionsHash => "ida96af0f171bb75f894a4013a6b3146a4397c58f11adb81a2b7cbea9314783a9";
```

Reply:
const would require to use the class name as a reference instead of the instance. I was to avoid doing that in case we need to mock that thing